ui/app/i18n/reports/locale_en.json

```diff
@@ -44,5 +44,6 @@
     "CHOOSE_ANSWER_FROM_DROPDOWN_LABEL": "Choose Answer",
     "DELETE_LABEL": "Delete",
     "EDIT_LABEL": "Edit",
-    "REPORT_DATE_RANGE": "Select Date Range"
+    "REPORT_DATE_RANGE": "Select Date Range",
+    "START_DATE_CANNOT_GREATER_THAN_STOP_DATE": "start date can not be greater than stop date"
```

Comment:
Can you rephrase this to "Later" instead of "Greater"?

_Start date cannot be later than the End Date_.